src/main/kotlin/net/ccbluex/liquidbounce/utils/block/BlockExtensions.kt

```diff
@@ -278,7 +281,11 @@ private fun handleActionsOnAccept(
     }
 
     if (onPlacementSuccess()) {
-        player.swingHand(hand)
+        if (!silentSwing) {
+            player.swingHand(hand)
+        } else {
+            mc.networkHandler!!.sendPacket(HandSwingC2SPacket(Hand.MAIN_HAND))
```

Comment:
Instead of using `mc.networkHandler!!.` use `network.sendPacket` instead. It is a short cut, but does the same.